source/api/image/3.0/change-log.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Image API 3.0 Change Log"
+title_override: "Changes for IIIF Image API Version 3.0"
+id: image-api-30-change-log
+layout: spec
+cssversion: 3
+tags: [specifications, image-api, change-log]
+major: 3
+minor: 0
+patch: 0
+pre: beta
+redirect_from:
+  - /api/image/3.0/change-log-30.html
+---
+
+This document is a companion to the [IIIF Image API Specification, Version 3.0][image30]. It describes the changes to the API specification made in this major release, including ones that are backwards incompatible with version 2.1.1, the previous version.
+
+## Table of Contents
+{:.no_toc}
+
+* Goes Here
+{:toc}
+
+
+## 1. Breaking Changes
+
+The following changes are backwards incompatible with version 2.1.1 of the Image API.
+
+### 1.1. Image Request Changes
+
+#### 1.1.1. Size `full` has been removed in favor of `max`
+
+Per the [deprecation warning][image21-full-dep] in the previous version, the value `full` is no longer allowed for the size parameter, `max` must be used instead. See issue [#678](https://github.com/IIIF/api/issues/678).
+
+#### 1.1.2. Canonical form of size parameter changed to _`w,h`_
+
+Per the [deprecation warning][image21-full-dep] and the [inconsistency warning][image21-size-inconsistency] about the size parameter in the previous version, the canonical form of the size parameter is now _`w,h`_ unless `max` is requested. This resolves the inconsistency between the server-preferred values in the `sizes` object, and the canonical form of the size parameter. In order to request preferred sizes, a client simply uses `width` and `height` values from `sizes` to build the _`w,h`_ size to request. See issues [#544](https://github.com/IIIF/api/issues/544) and [#678](https://github.com/IIIF/api/issues/678).
```

Comment:
> a client simply uses

"a client should use" -- who knows if it's simple or not?

Reply:
Hmmm... "simple" was intentional because one takes the value directly and just puts a comma between them.  Perhaps "should use ... unmodified"

Reply:
That would be fine w/ me.